cirq/circuits/qasm_output.py

```diff
@@ -148,12 +148,11 @@ def __init__(self,
                  header: str = '',
                  precision: int = 10,
                  version: str = '2.0') -> None:
-        self.operations = tuple(ops.flatten_op_tree(operations))
+        self.operations = tuple(ops.flatten_to_ops(operations))
```

Comment:
Not deprecated yet, but `flatten_op_tree` has a return type of `Iterator[Union[Operation, Moment]]` because it can optionally preserve moments; `flatten_to_ops` doesn't have this option so it returns `Iterator[Operation]` which is what we want here anyway and keeps mypy happy.